IntegrationAdapters/Services/SftpService.cs

```diff
@@ -10,9 +10,15 @@ namespace IntegrationAdapters.Services
 {
     public class SftpService : ISftpService
     {
+        public SftpConfig config = new SftpConfig
+        {
+            Host = "192.168.0.23",
+            Port = 22,
+            Username = "tester",
+            Password = "password"

```

Comment:
same here, you can make a class that contains all those constants as static fields and use them